quark/radiocontrast.py

```diff
@@ -60,7 +62,7 @@ def method_recursive_search(self, method_set, depth=1):
 
             self.method_recursive_search(self.apkinfo.lowerfunc(md[0]), depth)
 
-    def rule_generate(self):
+    def generate_rule(self, percentile_rank=0.2, web_editor=None):
         """
         Generate rules and export them to the output directory.
 
```

Comment:
The document for parameters `percentile_rank` and `web_editor` are missing. Could you help to fix it? Thanks!